code/modules/robotics/bot/secbot.dm

```diff
@@ -1152,16 +1156,29 @@ Report Arrests: <A href='?src=\ref[src];operation=report'>[report_arrests ? "On"
 		src.name = "helmet/signaler/prox sensor/robot arm assembly"
 		src.overlays += image('icons/obj/bots/aibots.dmi', "hs_arm")
 		qdel(W)
```

Comment:
```suggestion
    user.u_equip(W)
		qdel(W)
```

Reply:
I dont know what user.u_equip(W) is supposed to do here, but I added it anyway and it seems to work!